macros/sfdc_formula_view.sql

```diff
@@ -1,4 +1,4 @@
-{%- macro sfdc_formula_view(source_table, source_name='salesforce', reserved_table_name=source_table, fields_to_include=none, full_statement_version=true, materialization='view', using_quoted_identifiers=False) -%}
+{%- macro sfdc_formula_view(source_table, source_name='salesforce', reserved_table_name=source_table, fields_to_include=none, materialization='view', using_quoted_identifiers=False, full_statement_version=true) -%}
```

Comment:
It was actually just a result of my testing removing it and forgetting where it went. 😅 But I guess it could go last anyway since we're not really using it.